components/cartlist-card/index.js

```diff
@@ -107,12 +107,23 @@ export const CartCard = (props) => {
           display: inline-block;
           background-color: ${colors.pink.dark};
           font-size: 16px;
+          text-align: center;
         }
 
         @media only screen and (max-width: 600px) {
           .cartcard-container {
             flex-direction: column;
           }
+          .cartcard-button{
+            display: flex;
+            flex-direction: row;
```

Comment:
NIT: When we use `display: flex` by default the `flex-direction` is row, so even if we do not specify `flex-direction: row` it will automatically consider it as `row`

Reply:
@sumitd94 I have make it correct now. Please check.

Reply:
I see white space below drop down, that does not look good. Also the color for menu is very different can you change the color so that there is some kind of consistency.

Reply:
@Kratika0907  I have removed white spaces. Please review it. But I am confused about colors. Please clarify me about colors.

Desktop view
![Screenshot (248)](https://user-images.githubusercontent.com/70262783/104622684-92649000-56b7-11eb-8339-81a1bfdfbcd5.png)

Mobile view
![Screenshot (249)](https://user-images.githubusercontent.com/70262783/104622697-95f81700-56b7-11eb-9bbf-d47245d144d0.png)